db/models.py

```diff
@@ -1 +1,34 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255)
+    bonus = models.CharField(
+        verbose_name="This field describes what"
+                     "kind of bonus players can get from it."
+                     "In other words, this is "
+                     "a description of the bonus.",
+        max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255)
+    description = models.TextField(null=True)
+
+
+class Player(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(max_length=255, unique=False)
+    bio = models.TextField(
+        verbose_name="It stores a short description"
+                     "provided by a user about himself/herself.",
```

Comment:
The verbose_name for the 'bio' field is lengthy. A more concise version like 'User bio' could improve readability.